projects/core/src/lib/mask.ts

```diff
@@ -173,9 +186,7 @@ export class Maskito extends MaskHistory {
 
         const [from, to] = elementState.selection;
         const newPossibleValue =
-            elementState.value.slice(0, from) +
-            insertedText +
-            elementState.value.slice(to);
+            elementState.value.slice(0, from) + data + elementState.value.slice(to);
```

Comment:
What is the difference between `insertedText` and `data` in this case?

Reply:
* `data` - inserted characters BEFORE preprocessing (raw data from `Event.data`).
* `insertedText` - inserted characters AFTER preprocessing

We calculates `newPossibleValue` to decide if we have to prevent native event? Will natively typed character make mask valid (without programatic interference)?

It is especially useful for `RegExp`-mask (without fixed characters masks).
The majority of all typed characters can be inserted natively (without programatic value patching) ?